webview/src/RenderCanvas.vue

```diff
@@ -0,0 +1,1031 @@
+<script setup lang="ts">
```

Comment:
I wonder if it is possible to reuse these components from both playgroud website and this extension.

Reply:
It should be, though it'll require some extra work and probably a separate repository. Do you want me to work on that next?